packages/design-system/src/components/icon-button-with-menu.tsx

```diff
@@ -17,7 +17,7 @@ const StyledContent = styled(DropdownMenuPrimitive.Content, {
 
 const itemStyles = {
   all: "unset",
-  fontSize: "$2",
```

Comment:
Strange. Here 2 replaced with 3 ...

Reply:
that's correct, $2 in previous scale is 13px, in new one we don't have it so it's 12px which is $3